src/main/scala-2.12/org/apache/james/gatling/jmap/JmapAuthentication.scala

```diff
@@ -44,6 +45,6 @@ object JmapAuthentication {
     http(requestName)
       .post(endPoint)
       .header("Authorization", "${accessToken}")
-      .header("Accept", "application/json")
+      .header(JmapHttp.AcceptApplicationJson._1, JmapHttp.AcceptApplicationJson._2)
```

Comment:
I don't really like the use of tuple access for that, we should provide well named constants for that